### PR TITLE
Decoder related changes

### PR DESCRIPTION
**Pull request by LefterisJP**
- [Use a sequence counter to ensure tx event order](https://github.com/rotki/rotki/commit/802f170425920d126326dc82caa6bdfac9989b41)[](https://github.com/LefterisJP)

- [Ignore lgtm warning for missing-call-to-init](https://github.com/rotki/rotki/commit/a9874e14ca4357fc6582b49edf76950ab25443f9)[](https://github.com/LefterisJP)

- [CSVEXporter uses CustomizableDateMixin](https://github.com/rotki/rotki/commit/9244656bb429d619871064bce62900861021c75e)[](https://github.com/LefterisJP)

### CODE REVIEW

File: rotkehlchen/csv_exporter.py [hunk 1]
```diff
@@ -94,14 +95,15 @@ def _dict_to_csv_file(path: Path, dictionary_list: List) -> None:
...
-class CSVExporter():
+class CSVExporter(CustomizableDateMixin):
 
     def __init__(
             self,
             database: 'DBHandler',
             user_directory: Path,
             create_csv: bool,
     ):
+        super().__init__(database=self.database)
```
Comment (yabirgb) [inline rotkehlchen/csv_exporter.py:106]: You called `self.database` but that attribute is later initialized. Should be `database`

Comment (LefterisJP) [reply rotkehlchen/csv_exporter.py]: yes this is why the tests failed

**Review by yabirgb — APPROVED** [review]
Failing backend test is timeout from etherscan and the e2e seems to be a timeout adding a trade. Checked the video and doesn't seem to be anything wrong, just it taking too long.

**Comment by LefterisJP** [comment]
> Failing backend test is timeout from etherscan and the e2e seems to be a timeout adding a trade. Checked the video and doesn't seem to be anything wrong, just it taking too long.

yeah also checked and it seems fine.

**Pull request merged**